nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Update/UpdateWorkerTests.Sdk.cs

```diff
@@ -2998,7 +3034,7 @@ await TestUpdateForProject("Some.Package", "7.0.1", "13.0.1",
         [Theory]
         [InlineData("true")]
         [InlineData(null)]
-        public async Task UnresolvablePropertyDoesNotStopOtherUpdates(string variableValue)
+        public async Task ProjectWithWorkloadsShouldNotFail(string variableValue)
```

Comment:
Could you add this as a separate test?  I'd like to keep the existing test with an unresolvable property on its own.

Reply:
That's curious, I thought I did not change the original one. Will change for sure!

Reply:
@brettfo it's green again, thanks!